### PR TITLE
overhauls verb UI

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
overhauls verb UI; a list of verbs that got removed will be placed here later. Right now this cuts a lot of nearly unused verbs and moves preference verbs to a proper menu
complementary to #22224 

## Why It's Good For The Game
verb menus are a mess and a lot of these need shooting, being able to CTRL-F through the pref menu is good and admin menus are an absolute mess.
## Images of changes
there we go

## Testing
Clicked on verbs and other menu stuff

## Changelog
:cl:
tweak: The verb panel has been overhauled
/:cl:

### CODE REVIEW

File: code/__DEFINES/preferences_defines.dm [hunk 1]
```diff
@@ -77,7 +77,21 @@
 #error toggles_2 bitflag over 16777215. Please make an issue report and postpone the feature you are working on.
 #endif
 
-
+// Preftoggle defines
+#define PREFTOGGLE_GENERAL		1
```
Comment (Contrabang) [inline code/__DEFINES/preferences_defines.dm:81]: Not gonna get too indepth with a review, but you should probably preface some of these defines, something like category may be good
```suggestion
#define PREFTOGGLE_CATEGORY_GENERAL		1
```

**Comment by Henri215** [comment]
Whats left to do here?

**Comment by GDNgit** [comment]
> Whats left to do here?

Pretty much just UI stuff along with removing the last couple verbs. I'm probably gonna finish this fully post TGUI 4 to benefit from the tgui statpanel

**Review by S34NW — CHANGES_REQUESTED** [review]
TGUI update is in, please deconflict and update this.

**Comment by GDNgit** [comment]
pr has been split in two parts

**Pull request closed without merging**